src/main/java/uk/gov/companieshouse/api/strikeoffobjections/config/ApplicationConfig.java

```diff
@@ -3,7 +3,9 @@
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
```

Comment:
Please check the sonar report before putting code up for review, this import is unused

Reply:
Seen that - taken it out.